.github/workflows/test-external-libs.yml

```diff
@@ -0,0 +1,64 @@
+name: External Libraries
```

Comment:
Curious, why are these tests in a separate configuration? Shouldn't these also depend on the preceding linting/testing?

Reply:
Technically these libs aren't apart of the plugin so these tests shouldn't be impeded by the plugin's lint/tests jobs (unless I'm missing something here).

/cc @schlessera

Reply:
I'm currently working on extracting these libraries out (https://github.com/ampproject/amp-toolbox-php), so we shouldn't waste too much thought on these. I will rebuild GHA testing on that other repo.

Reply:
The tests are already running on the other repository now:
![Image 2020-10-29 at 5 16 52 PM](https://user-images.githubusercontent.com/83631/97601602-a37ff800-1a0a-11eb-8fa8-141676863af6.png)

So there's no need to do any work on them here anymore.

Reply:
Awesome! WIll remove the workflow then.